src/python/foglamp/translators/omf_translator.py

```diff
@@ -936,6 +941,13 @@ def omf_object_creation():
         _log.error(message)
         raise Exception(message)
 
+async def _update_statistics():
+    global _num_readings
+    global _num_discarded_readings
```

Comment:
Where are the variables _num_readings and _num_discarded_readings used? In the next few lines we are using _num_sent and _num_unsent.